client/src/components/CustomDateInput.js

```diff
@@ -75,7 +75,7 @@ const CustomDateInput = ({
       placeholder={inputFormat}
       maxDate={maxDate}
       minDate={moment().subtract(100, "years").startOf("year").toDate()}
-      canClearSelection={false}
+      canClearSelection
```

Comment:
Why not turn this into a prop (default `false`), and only pass `true` in places where clearing the date input is allowed?

Reply:
You are correct, I will set it as false by default, and I will be passing prop where clearing is allowed.